pkg/infra/metrics/metrics.go

```diff
@@ -201,6 +201,9 @@ var (
 
 	// MStatTotalPublicDashboards is a metric total amount of public dashboards
 	MStatTotalPublicDashboards prometheus.Gauge
+
```

Comment:
are those changes supposed to be in this PR?

Reply:
Sweet mother of Type Script, hell no! I messed something up, but should be good now. These changes were supposed to go to https://github.com/grafana/grafana/pull/66021